authors/apps/profiles/migrations/0002_auto_20190409_0901.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.7 on 2019-04-09 09:01
```

Comment:
This migration isn't related to this task. Consequently I don't think we should have it in this PR